addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/handler/HeliosHandler221.java

```diff
@@ -201,6 +202,17 @@ public boolean verify(String hostname, javax.net.ssl.SSLSession sslSession) {
 
             JsonObject jsonObject = parser.parse(response.readEntity(String.class)).getAsJsonObject();
 
+            if (logger.isTraceEnabled()) {
+                logger.trace("initialize() Request : {}", systemTarget.resolveTemplate("ip", ipAddress)
+                        .resolveTemplate("cmd", INFO).getUri().toASCIIString());
+                if (jsonObject.get("success").toString().equals("true")) {
+                    logger.trace("initialize() Response: {}", jsonObject.get("result").toString());
```

Comment:
Please remove all calls to `toString()` as this is done by the logger automatically.